locales/base/translation.json

```
@@ -1293,7 +1293,9 @@
     "selectedPaymentOption": "Selected payment option",
     "addFiatAccountSuccess": "Your account has been linked to Valora.",
     "addFiatAccountResourceExist": "You have already linked this account to Valora.",
-    "addFiatAccountFailed": "Your account was not linked, please try again or contact support."
+    "addFiatAccountFailed": "Your account was not linked, please try again or contact support.",
+    "selectItem": "Select an item...",
```

just sounds slightly better to me
```suggestion
    "selectItem": "Select an option...",
```